composer.json

```diff
@@ -21,7 +21,7 @@
     },
     "require": {
         "php": "^7",
-        "ext-json": "^1",
+        "ext-json": "*",
```

Comment:
`^1|^7` ?

Reply:
A few libraries suggested that when this initially became an issue, but their master/main branches reference `*` as the version constraint, so I can only assume there is some hidden gotcha to that kind of non linear constraint.